packages/medusa/src/repositories/customer-group.ts

```diff
@@ -1,5 +1,31 @@
-import { EntityRepository, Repository } from "typeorm"
+import { MedusaError } from "medusa-core-utils"
+import { EntityRepository, In, Repository } from "typeorm"
 import { CustomerGroup } from "../models/customer-group"
+import { CustomerBatchIds } from "../types/customer-groups"
 
 @EntityRepository(CustomerGroup)
-export class CustomerGroupRepository extends Repository<CustomerGroup> {}
+export class CustomerGroupRepository extends Repository<CustomerGroup> {
+  async deleteCustomerBatch(
+    groupId: string,
+    customerIds: CustomerBatchIds[]
+  ): Promise<CustomerGroup | undefined> {
+    const customerGroup = await this.findOne(groupId)
+    if (!customerGroup) {
+      throw new MedusaError(
+        MedusaError.Types.NOT_FOUND,
+        `CustomerGroup with id ${groupId} was not found`
+      )
+    }
```

Comment:
This is not necessary here; just call the delete query - for reference: https://github.com/medusajs/medusa/blob/b11fd4f300e330fa655d55d643a856f8689bbdb3/packages/medusa/src/repositories/tax-rate.ts#L185